src/test/java/net/datafaker/sequence/FakeStreamTest.java

```diff
@@ -128,7 +128,7 @@ void checkWrongArguments() {
                 .maxLen(5)
                 .generate())
             .isInstanceOf(IllegalArgumentException.class)
-            .hasMessage("Max length must be not less than min length and not negative");
+            .hasMessage("Max length (5) must be not less than min length (10) and not negative");
```

Comment:
wouldn't it be better to extract it into vars?

Reply:
I would say it makes this test more complex.
These values are pretty near and well-readable. After introducing variables, it will be harder to read this test.